app/src/main/java/com/example/wordsapp/LetterListFragment.kt

```diff
@@ -0,0 +1,91 @@
+package com.example.wordsapp
+
+import android.os.Bundle
+import android.view.*
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.wordsapp.databinding.FragmentLetterListBinding
+
+class LetterListFragment : Fragment() {
+
+    private var _binding: FragmentLetterListBinding? = null
+
+    private val binding get() = _binding!!
+
+    private lateinit var recyclerView: RecyclerView
+
+    private var isLinearLayoutManager = true
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
```

Comment:
Please check kotlin coding conventions.

https://kotlinlang.org/docs/reference/coding-conventions.html
Classes with longer headers should be formatted so that each primary constructor parameter is in a separate line with indentation. Also, the closing parenthesis should be on a new line. If we use inheritance, then the superclass constructor call or list of implemented interfaces should be located on the same line as the parenthesis